docs/tutorials/contributing/strategy/writing_the_new_strategy.rst

```diff
@@ -7,7 +7,7 @@ Identify a new strategy
 The library has a large number of strategies::
 
     >>> import axelrod as axl
-    >>> len(axl.strategies)
+    >>> len(axl.strategies) # doctest: +SKIP
     103
 
 If you're not sure if you have a strategy that has already been implemented
```

Comment:
> Could we have an update script that corrects the value somehow?

This can go wrong if we get lazy. I think that it's best to just remove it all together. As you say the diagrams show the number of strategies :) :tada: